### DISCUSSION TITLE
Where is RustDesk client password stored?

### DISCUSSION THREAD

**Discussion by emammadov** [rustdesk/rustdesk#9439] [General]
Hello,
We have a Windows image for general purposes. In Windows Audit Mode I have installed a RustDesk client, configured the settings and assigned a password. In Windows Sysprep is ready, I can not connect a RustDesk client from the other workstations, it gives a message of wrong password. I assign the password again, then I can connect. That's why I would like to know where the RustDesk client password is stored. Could you please share this information with me? Thanks.

**Comment by xlionjuan** [comment]
`%appdata%`

**Comment by mslinn** [comment]
`%AppData%\RustDesk\config\peers`

**Reply by Yrij-Zhavoronkov** [reply]
It stores it there, maybe it stores it, but how can I use this information? 
I reinstalled Windows and moved this folder. As a result, I have a list of peers, but for each one, when connecting, the error "incorrect password?" is displayed. 
Why is this and how can I fix it?

**Comment by mslinn** [comment]
I suspect that a client-side string (perhaps the computer's network name) is used as a [salt](https://auth0.com/blog/adding-salt-to-hashing-a-better-way-to-store-passwords/) in the password hashing process. If true, then unless you can specify the hash (unlikely) the passwords will have to be specified again.

Just a guess.
